### DISCUSSION TITLE
Removing .html extension from URL

### DISCUSSION THREAD

**Discussion by NicholasGannon** [community/community#137956] [Programming Help]
### Body

When developing websites: what's the best practice when it comes to **removing the .html extension** from URL's?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Y3llow45** [comment]
From your repos i saw that you use live server.
Create `live-server.json` and put this inside
`{
  "mount": [
    { "path": "/", "dir": "." }
  ],
  "rewrites": [
    { "source": "**", "destination": "index.html" }
  ]
}`
Modify your `package.json`: 
```
"scripts": {
  "dev": "live-server --config=live-server.json"
}
```

**Reply by NicholasGannon** [reply]
I believe I found my solution by nesting index.html files within a directory - Thank you for the information though!

**Comment by localnetwork** [comment]
It looks like you've resolved your issue. However, typically, you can include an index.html file within your folders, like this:

project-folder
 ├── index.html (homepage)
 ├── about-us
 │   └── index.html
 └── contact-us
     └── index.html

**Reply by PIYUSH1525** [reply]
does this works ??

**Reply by NicholasGannon** [reply]
Yes! This appears to work on my end: Nesting index.html within a directory!

**Reply by localnetwork** [reply]
> does this works ??

@PIYUSH1525 yes, it will work.
